mapreduce/metrics.csv

```diff
@@ -0,0 +1,131 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+mapreduce.job.elapsed_time.max,gauge,,millisecond,,The max elapsed time since the application started,0,mapreduce,jb time mx
+mapreduce.job.elapsed_time.avg,gauge,,millisecond,,The average elapsed time since the application started,0,mapreduce,jb time avg
+mapreduce.job.elapsed_time.median,gauge,,millisecond,,The median elapsed time since the application started,0,mapreduce,jb time mx med
+mapreduce.job.elapsed_time.95percentile,gauge,,millisecond,,The 95th percentile elapsed time since the application started,0,mapreduce,jb time mx 95
+mapreduce.job.elapsed_time.count,rate,,,,The number of times the elapsed time was sampled,0,mapreduce,jb time mx cnt
+mapreduce.job.maps_total.max,gauge,,task,,The max max total number of maps,0,mapreduce,jb mps mx
+mapreduce.job.maps_total.avg,gauge,,task,,The avg total number of maps,0,mapreduce,jb mps avg
+mapreduce.job.maps_total.median,gauge,,task,,The median total number of maps,0,mapreduce,jb mps med
+mapreduce.job.maps_total95percentile,gauge,,task,,The 95th percentile total number of maps,0,mapreduce,jb mps 95
+mapreduce.job.maps_total.count,rate,,,,The number of times the max total was sampled,0,mapreduce,jb mps cnt
+mapreduce.job.maps_completed.max,gauge,,task,,The max number of completed maps,0,mapreduce,jb mp com mx
+mapreduce.job.maps_completed.avg,gauge,,task,,The average number of completed maps,0,mapreduce,jb mp com avg
+mapreduce.job.maps_completed.median,gauge,,task,,The median number of completed maps,0,mapreduce,jb mp com med
+mapreduce.job.maps_completed.95percentile,gauge,,task,,The 95th percentile number of completed maps,0,mapreduce,jb mp com 95
+mapreduce.job.maps_completed.count,rate,,,,The number of times the number of completed maps were sampled,0,mapreduce,jb mp com cnt
+mapreduce.job.reduces_total.max,gauge,,task,,The max total number of reduces,0,mapreduce,jb red mx
+mapreduce.job.reduces_total.avg,gauge,,task,,The average total number of reduces,0,mapreduce,jb red avg
+mapreduce.job.reduces_total.median,gauge,,task,,The median total number of reduces,0,mapreduce,jb red med
+mapreduce.job.reduces_total.95percentile,gauge,,task,,The 95th percentile total number of reduces,0,mapreduce,jb red 95
+mapreduce.job.reduces_total.count,rate,,,,The number of times the total number of reduces were sampled,0,mapreduce,jb red cnt
+mapreduce.job.reduces_completed.max,gauge,,task,,The max number of completed reduces,0,mapreduce,jb red comp mx
+mapreduce.job.reduces_completed.avg,gauge,,task,,The average number of completed reduces,0,mapreduce,jb red comp avg
+mapreduce.job.reduces_completed.median,gauge,,task,,The median number of completed reduces,0,mapreduce,jb red comp med
+mapreduce.job.reduces_completed.95percentile,gauge,,task,,The 9th percentile number of completed reduces,0,mapreduce,jb red comp 95

```

Comment:
small typo: `9th percentile` -> `95th percentile`

also applies to the rest of this file ;)